src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -1268,6 +1269,15 @@ mobile_apis::VideoStreamingState::eType StateControllerImpl::CalcVideoState(
     ApplicationSharedPtr app,
     const mobile_apis::HMILevel::eType hmi_level) const {
   auto state = mobile_apis::VideoStreamingState::NOT_STREAMABLE;
+
+  if (app->webengine_projection_enabled()) {
+    LOG4CXX_DEBUG(logger_,
+                  "Calculated video state of app "
+                      << app->app_id() << " for " << hmi_level
+                      << " HMI level is " << state);
+    return state;
+  }
+
   if (IsStreamableHMILevel(hmi_level) && app->IsVideoApplication()) {
```

Comment:
@AByzhynar could you please consider a suggestion
```suggestion

  if (IsStreamableHMILevel(hmi_level) && app->IsVideoApplication() && !app->webengine_projection_enabled()) {
```

Reply:
@mvorobio Don't see any sense. app->IsVideoApplication() already contains this logic. This check `!app->webengine_projection_enabled()` will never be executed